Source/Tools/LinuxEmulation/LinuxSyscalls/x32/Syscalls.h

```diff
@@ -48,16 +47,9 @@ class x32SyscallHandler final : public FEX::HLE::SyscallHandler {
                           const fextl::string& TraceFormatString,
 #endif
                           void* SyscallHandler, int ArgumentCount) override {
-    auto& Def = Definitions.at(SyscallNumber);
+    LOGMAN_THROW_A_FMT(SyscallNumber < Definitions.size(), "Trying to add syscall number way beyond extent!");
+    auto& Def = Definitions[SyscallNumber];
```

Comment:
I'm not convinced the saved cost of a bounds check in Release mode justifies the added code noise here. Was there any particular use case you profiled to come up with this optimization?

Reply:
Removed.